src/py/_pyodide/_importhook.py

```diff
@@ -150,15 +154,7 @@ class UnvendoredStdlibFinder(MetaPathFinder):
     def __init__(self) -> None:
         # `test`` is not a stdlib module, but we unvendors in anyway.
         self.stdlibs = sys.stdlib_module_names | {"test"}
-
-        # TODO: put list of unvendored stdlibs to somewhere else?
-        self.unvendored_stdlibs = {
-            "distutils",
-            "test",
-            "ssl",
-            "lzma",
-            "sqlite3",
-        } & self.stdlibs
+        self.unvendored_stdlibs = UNVENDORED_STDLIBS_AND_TEST & self.stdlibs
```

Comment:
Could you add a comment to explain the point of this? Shouldn't everything in `UNVENDORED_STDLIBS` be in `self.stdlibs`?

Reply:
> Shouldn't everything in UNVENDORED_STDLIBS be in self.stdlibs?

Yes UNVENDORED_STDLIBS should be a subset of stdlibs. I just wanted to make sure that we don't add non-stdlibs by mistake.

Reply:
Could we make that into an assertion then?

Reply:
Sounds good.